src/agents/supervisor.py

```
@@ -0,0 +1,102 @@
+    Attributes:
+        question: question
+        generation: LLM generation
+        document: retrieved document
+    """
+
+    question: str # 사용자의 질문
+    generation: str # LLM의 생성 답변
+    document: str # 검색된 문서
+    messages: Annotated[list, add_messages] # 메시지 히스토리 저장
+
+from pydantic import BaseModel, Field
```

Ruff (0.14.2)

Redefinition of unused `BaseModel` from line 7

Remove definition: `BaseModel`

(F811)